test/reload/run-scratch.ts

```diff
@@ -1,10 +0,0 @@
-const http = require("http");
```

Comment:
This file gets overwitten in tests. Its absence from `.gitignore` was an omission.

Reply:
Whoops -- thanks